.circleci/config.yml

```diff
@@ -39,39 +39,46 @@ jobs:
   doctest:
     <<: *common
     docker:
-      - image: circleci/python:3.5
+      - image: circleci/python:3.6
```

Comment:
Also, I just sent https://github.com/ethereum/ethereum-python-project-template/pull/19 so we don't have to do that again for new projects.